src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+/**
+ * @author user
+ * This class is an abstract class for Email, Phone, Address
+ */
```

Comment:
The interface does not need to be limited to just those contacts and could be used for other possible contacts you want to implement in the future.